package/gluon-hoodselector/luasrc/usr/lib/lua/hoodselector/util.lua

```diff
@@ -0,0 +1,83 @@
+local unistd = require('posix.unistd')
+local util = require 'gluon.util'
+local math_polygon = require('math-polygon')
+local json = require 'jsonc'
+local uci = require('simple-uci').cursor()
+local site = require 'gluon.site'
+local logger = require('posix.syslog')
```

Comment:
Question: Is there a difference between `require 'foo'` and `require('foo')`?

Reply:
seems no difference.
https://www.lua.org/pil/8.1.html

Reply:
Then pick one or the other, but please don't mix :)